tests/rwlock.rs

```diff
@@ -120,7 +120,8 @@ fn get_mut() {
     assert_eq!(lock.into_inner(), 20);
 }
 
-#[cfg(not(target_family = "wasm"))]
+// Miri bug; this works when async is replaced with blocking
```

Comment:
nitpick: Best if we have the URL to the relevant issue here, if there is one.

Reply:
I'd like to verify that this *is* a MIRI bug before I file one

Reply:
Fair enough. I guess it shouldn't block this PR and release nevertheless, since the even if it is our bug, it's in event-listener crate and will need to be fixed there?